go-auth/google/config.go

```diff
@@ -0,0 +1,156 @@
+package google_auth
```

Comment:
I debated internally over where to put this. It should be able to be re-used amongst google connectors (sheets, etc.) for authorization with service account & oauth options, so it seemed like having it somewhere it could be accessible to multiple connectors would make sense. In a go monorepo I'd put it in `pkg` but since that's not the case here this is the best I could come up with. Very open to alternatives though.